src/components/Navigation.js

```diff
@@ -5,27 +5,51 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { useRef } from 'react';
+import React, { useRef, useState } from 'react';
 import { NavLink, Link, useHistory } from 'react-router-dom';
 import hathorLib from '@hathor/wallet-lib';
 import { useFlag } from '@unleash/proxy-client-react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNewUiEnabled } from '../hooks';
 import logo from '../assets/images/hathor-white-logo.png';
+import moon from '../assets/images/moon.svg';
+import sun from '../assets/images/sun-dark.svg';
+import { ReactComponent as NewLogo } from '../assets/images/new-logo.svg';
```

Comment:
When importing an SVG as a React component, it's easier to modify properties like fill and stroke to match the theme's styling. However, when using the SVG as a background, we often need to import both versions, since we can't dynamically adjust the SVG properties in that context.